src/tools/rados/rados.cc

```diff
@@ -373,6 +428,69 @@ static int do_copy_pool(Rados& rados, const char *src_pool, const char *target_p
   return 0;
 }
 
+#ifndef USE_LIBIRADOSSTRIPER
```

Comment:
where is `USE_LIBIRADOSSTRIPER` defined?

Reply:
It is not, that's at typo. Good catch, thank you!